docs/extend/modular-resources/examples/triton.md

```diff
@@ -0,0 +1,179 @@
+| `backend_directory` | string | Optional | A container side path to the TritonServer "backend" directory. You normally do not need to override this; the build will set it to the backend directory of the Triton Server installation in the container. You may set it if you wish to use a different set of backends. |
+| `model_version` | int | Optional | The version of the model to be loaded. If not specified, the module will use the newest version of the model named by model_name. <br> Default: `-1` (newest) |
+| `preferred_input_memory_type` | string | Optional | One of `cpu`, `cpu-pinned`, or `gpu`. This controlls the type of memory that will be allocated by the module for input tensors. If not specified, this will default to `cpu` if no CUDA-capable devices are detected at runtime, or to `gpu` if CUDA-capable devices are found.|
+| `preferred_input_memory_type_id` | int | Optional | CUDA identifier on which to allocate gpu or cpu-pinned input tensors. You probably don't need to change this unless you have multiple GPUs. <br> Default: `0` (first device) |
+| `tensor_name_remappings` | string | Optional | Provides two dictionaries under the `input` and `output` keys that rename the models' tensors. Other Viam services, like the [vision service](/services/vision/) may expect to work with tensors with particular names. Use this map to rename the tensors from the loaded model as needed to meet those requirements. <br> Default: `{}` |
+
+An example minimal configuration would look like this, within your robot's `"services"` array:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "type": "mlmodel",
+  "attributes": {
+    "model_name": "efficientdet-lite4-detection",
+    "model_repository_path": "/path/to/.viam/triton/repository"
+  },
+  "model": "viam:mlmodelservice:triton",
+  "name": "mlmodel-effdet-triton"
+}
+```
+
+An example detailed configuration with optional parameters specified would look like this:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "type": "mlmodel",
+  "attributes": {
+    "backend_directory": "/opt/tritonserver/backends",
+    "model_name": "efficientdet-lite4-detection",
+    "model_version": 1,
+    "model_repository_path": "/path/to/.viam/triton/repository",
+    "preferred_input_memory_type_id": 0,
+    "preferred_input_memory_type": "gpu",
+    "tensor_name_remappings": {
+      "outputs": {
+        "output_3": "n_detections",
+        "output_0": "location",
+        "output_1": "score",
+        "output_2": "category"
+      },
+      "inputs": {
+        "images": "image"
+      }
+    }
+  },
+  "model": "viam:mlmodelservice:triton",
+  "name": "mlmodel-effdet-triton"
+}
+```
+
+## Next steps: configure a vision service
+
+Now, use the `viam:mlmodelservice:triton` modular service to deploy MLModels to your robot.
```

Comment:
This sentence seems out of place.

Reply:
Does `make use of` sound better? I guess deploy is misleading as it doesn't happen here. Can just remove sentence outright but want some kind of intro